### PR TITLE
Add new HJT command

### PR DESCRIPTION
**Pull request by urielsalis**

### CODE REVIEW

File: handlers/hjt/hjt.go [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+	db, err := database.Get()
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Failed to connect to database")
+		logger.Err().Printf("Failed to connect to database\n%s", err)
+		return
+	}
+
+	var data []hjt
+	err = db.Find(&data).Error
```
Comment (LordRalex) [inline handlers/hjt/hjt.go]: Why aren't we just using the DB to handle this concept? You can do a SELECT with a LIKE. This is going to be slow and eat way more memory than it should.

Comment (urielsalis) [reply handlers/hjt/hjt.go]: I thought LIKE only checked if what you are passing its contained in database, this has to be the opposite way(from our big chunk of text, we want to see what in database is contained there)

I suck at SQL so if you have any suggestions please tell me :D

Comment (LordRalex) [reply handlers/hjt/hjt.go]: You can do the reverse:

'myteststring' LIKE '*' + column + '*'

Comment (urielsalis) [reply handlers/hjt/hjt.go]: PErfect, will try. Can you make sure the query limit in mysql is not too low?

Comment (urielsalis) [reply handlers/hjt/hjt.go]: Done

File: handlers/hjt/hjt.go [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+	db, err := database.Get()
+	if err != nil {
+		_, err = ds.ChannelMessageSend(c.ID, "Failed to connect to database")
+		logger.Err().Printf("Failed to connect to database\n%s", err)
+		return
+	}
+
+	var data []hjt
+	var values []string
+	err = db.Find(&data).Where("? LIKE CONCAT('%', LOWER(name) ,'%')", content).Pluck("value", &values).Error
```
Comment (LordRalex) [inline handlers/hjt/hjt.go]: This now has a pointless struct, you can use .Table to specify the table and not need your struct definition (also means you control the table name)

Comment (urielsalis) [reply handlers/hjt/hjt.go]: done

**Pull request closed without merging**